parse/index.js

```diff
@@ -354,6 +354,32 @@ export default ({
     parseProps(i, block, last)
     lookForFonts(block)
     lookForMultiples(block)
+
+    let data = getData(block.properties.find((p) => p.name === 'data'))
+    if (data) {
+      let format = getDataFormat(
+        block.properties.find((p) => p.name === 'format')
+      )
+      let validate = getDataValidate(
+        block.properties.find((p) => p.name === 'validate')
+      )
+
+      block.data = { ...data, format, validate }
+    }
```

Comment:
let's add this block into the `flowProp` check for now until we have the flow implemented in the DS